### ISSUE TITLE
Anki Crashes (M1) PyQT 6

### ISSUE THREAD

**Issue by MedicAyman** [ankitects/anki#1883]
**Version ⁨2.1.52 (ab1c2395)⁩, Python 3.9.7 Qt 6.3.0 PyQt 6.3.0**
Macbook Pro M1 8Gb, MacOS Monterey 2.4 (21F79)

Happened twice so far during the review, I do not know to reproduce but this happens during reviews where the app crashes suddenly. The following diagnostic report is given . Not sure if it's because of PyQT6 or a bad interaction between the newer version or add-ons (no problem was present on 2.4X, PyQT5). 

I will provide the debug info at the end.

Diagnostic report
[Diagnostic report.txt](https://github.com/ankitects/anki/files/8749079/Diagnostic.report.txt)

Debug Info: 
[Debug Info.txt](https://github.com/ankitects/anki/files/8749080/Debug.Info.txt)

**Comment by dae** [comment]
Possibly related to #1879. Can you reproduce it with all add-ons disabled?

**Comment by dae** [comment]
What accessibility features do you have enabled? https://forums.ankiweb.net/t/change-note-type-anki-for-mac-2-1-45-crash/11937/6

**Comment by MedicAyman** [comment]
> What accessibility features do you have enabled? https://forums.ankiweb.net/t/change-note-type-anki-for-mac-2-1-45-crash/11937/6

None of the accessibility settings are enabled. I have only one app (rectangle) that has access to accessibility to work. But I do not it is related.



> Possibly related to #1879. Can you reproduce it with all add-ons disabled? No, but I can't also also reproduce with the same add-ons on.

**Comment by dae** [comment]
Does rc2 help? https://betas.ankiweb.net/anki2.1.53.html#release-candidate-2

**Comment by MedicAyman** [comment]
> Does rc2 help? https://betas.ankiweb.net/anki2.1.53.html#release-candidate-2

Was fixed, I don't know if I should make another issue about, kinda still relates to qt6 but there are scroll bars always present (Same laptop, macbook pro M1). Sometimes they appear sometime not. 
Thanks!
<img width="1439" alt="Screenshot 2022-05-26 at 09 35 29" src="https://user-images.githubusercontent.com/62990440/170431595-1b3d56f1-79f4-4fe5-8055-12711773e264.png">

**Comment by dae** [comment]
My guess is you have an add-on installed that has changed the button appearance, causing the scrollbar.

**Comment by MedicAyman** [comment]
> My guess is you have an add-on installed that has changed the button appearance, causing the scrollbar.

Fixed. Thanks!

**Comment by Neosaurus** [comment]
> Does rc2 help? https://betas.ankiweb.net/anki2.1.53.html#release-candidate-2

Tentatively fixed for me (cannot reproduce in my reproducible instance). Thanks! WIll update if anything changes

**Comment by blackace72** [comment]
```
Anki 2.1.53 (f6ecddef) Python 3.9.7 Qt 6.3.0 PyQt 6.3.0
Platform: Mac 12.3.1
Flags: frz=True ao=False sv=3
Add-ons, last update check: 2022-05-27 00:04:38
Add-ons active: None
```

How to reproduce:
1. Select a deck from Deck List screen
2. Press `o` for options screen
3. Press `esc`
4. Press `d` to return to Deck List Screen
5. Repeat steps 1-4 for different decks until crash

**Comment by dae** [comment]
Please test with rc2.

**Comment by blackace72** [comment]
Immediate improvement; can't reproduce so far. Thanks!

**Comment by dae** [comment]
2.1.53 is out today with this fix (it's identical to 2.1.53rc2). For anyone still experiencing crashes after upgrading, please post on the forums so we can triage there.
